src/marketing/email-reminder-rules-create.md

```diff
@@ -90,8 +90,8 @@ Email reminders might promote a cart price rule with, or without, a coupon. A ca
 
 |Source|Trigger|
 |--- |--- |
-|Wish List|Number of days abandoned<br/>Sharing<br/>Number of items<br/>Specific items|
-|Shopping Cart|Number of days abandoned<br/>Specific coupon code applied<br/>Line items quantity<br/>Items quantity<br/>Total<br/>Virtual items<br/>Specific Items|
+|Wish List|Conditions Combination<br/>Sharing<br/>Number of Items<br/>Items Sublection|
```

Comment:
Is that 'Items Subsection' or 'Items Selection'? Both lines.

Reply:
Oh, thank you, it is `subselection` 😊